config/routes.rb

```diff
@@ -30,7 +30,7 @@
     get "/still-working", to: "still_working#show"
     post "/still-working", to: "still_working#submit"
 
-    # Question: Are you worried about going into work because you live with someone vulnerable to coronavirus?
+    # Question: Are you worried about going into work because you live with someone vulnerable
```

Comment:
I'm not sure this was intentional?

Reply:
ah no, error when rebasing! Thanks, have amended.